### DISCUSSION TITLE
Is there a way to define what is silent?

### DISCUSSION THREAD

**Discussion by mwwmrs** [WyattBlue/auto-editor#398] [1. Help] [answered]
Is there a command to define what is silent parts? Right now it's cutting a lot of my words and some sentences, I don't want to increase mic volume because then it will pick up more unwanted noises that are harder to clear out. I don't know at what -db it's set right now, but i would like to make it around -28db for silent part, is there a command to do that?
I might be blind, but I don't think i saw one on the guide list. Thanks for the help in advance.

**Comment by WyattBlue** [comment, marked as answer]
The `--edit` options determines what is considered 'silent' and 'loud'.

This command should work for you:
```
auto-editor example.mp4 --edit audio:threshold=-28dB
```

**Reply by mwwmrs** [reply]
Thank you!

**Comment by aylardananil** [comment]
What is the default threshold value of auto-editor? 28?
